### PR TITLE
Actually remove xenobio & update maps

### PR DESCRIPTION
**Pull request by Poojawa**
This has #5893 's changes, in addition to removing the xenobio consoles and kill chambers from roundstart. It is possible convert the rooms back if you're clever about it.

This is based upon the tentatively positive feedback I received regarding the brief time that the console removal change was actually in effect. Leading me to believe that despite certain detractor's vitriol, it wasn't actually the end of the department at all. People were having more fun interacting with slimes and being better at monkey wrangling. 

This also is a better organized one, and doesn't have missing power wires in Metastation's xenobio. 

I'd much rather this one be merged, but to appease *certain* individuals, it's open to discussion.

### CODE REVIEW

**Comment by yourdoom9898** [comment]
Despite what Bhijin says, I have seen several discussions in Discord after the merge from different people saying that the new change wasn't good. It creates boring tedium for the sake of creating boring tedium, while not actually making xenobio more challenging in any significant way, besides for saying "hello" to not get eaten.

**Comment by GuppyLaxx** [comment]
Yet again, why? Not even rewarding in the least, an unneeded change in general, it's an annoyance for players, with pretty much zero explanation behind why it should be removed in the first place, positive feedback on the time it was gone my ass, it was a full shitstorm, or of course, you're back to listening to only the *positive* shit, and nice of you to say that in specific at the end.
This doesn't end xenobio at all, but it's unneeded as shit, and pretty much no one has said that it was gonna/it ended xenobio.

**Comment by Poojawa** [comment]
Yes, I'm aware of your bias against this, Yourdoom, and you're one of the noted detractors. 

And yet again, it seems my explanations are discarded. wew

**Comment by CameronWoof** [comment]
I believe that one of the sticking points was monkeys being massive jerks. Do you think you could fit in the monkeycube delay change you discussed into this PR?

**Comment by Poojawa** [comment]
Easy enough of an addition ~

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Knightfall5** [comment]
I've seen xenobiology used once since consoles were removed, and the person quit before they got a single T3 slime. Before, it was common to see xenobiology get used every other round or so, and people would stay in there for a good portion of the round messing around with it. Now? It's back to the barren, unused wasteland it was before consoles.

Consoles were added because xenobiology is boring and tedious without them, it was the least used part of the entire station before they existed. I've played every xenobio system since Metroids, and it just isn't fun to do at all without the consoles.

**Comment by Poojawa** [comment]
People are just not hard workers anymore for the most part, especially /tg/ base players. Yes it's tedious, but for the absolutely insane power-ups you can get, it makes sense.

Mining has to work for Megafauna rewards, even though by now most people are being taught how to cheese the patterns and ai to make it that much easier. That'll be fixed if GOAP goes from pipe dream to workable.

point values were adjusted for R&D, the first thing people wanted to do was mitigate the increase by giving themselves a hard, major point income boost that lessens the whole point to the point values. And makes everything relatively cheaper to finish out. 

Meteors blow up some of the station, people's first reaction is 'DAMAGE IS IRREPARABLE, CALL THE SHUTTLE!' when it's like, a couple of walls and windows at most. in each direction of the station. Rebuilding takes effort.

**Comment by Knightfall5** [comment]
People want their effort to provide them with fun. Fighting megafauna is fun, wrangling monkeys and slimes is boring. And it's not a TG playerbase issue, an actual high RP server, Baystation, has a similar Xenobiology system. And it's almost never used. Same on Polaris, same on Virgo. And the Polaris/Virgo xenobiology system is MUCH more interactive than the console-less version here is.

**Comment by AmatsuDF** [comment]
While perhaps needed on some level, there's far better suggestions for this to be handled rather then a hard 'this is late game only'. I recall seeing a suggestion on Discord that would make it so the console starts as a camera, and as better and better parts are installed, it regains its old functions over time. THAT would be superior to this by far.

**Comment by JogBoba** [comment]
I believe that if the console is to be removed from the map, it needs to be easier to access via techwebs than it is currently. I also do not think total removal of the kill room is the right thing to do, when you can simply make it so that the room has to be manually filled and cooled via pumping in cooled gas from a canister. As it stands however, especially with the low point gain, this change is not one that will be well received and should be testmerged at most.

**Comment by JogBoba** [comment]
Of another note, xenobiology doesn’t have nearly the rewards of megafauna, and fighting megafauna is an interactive and fun experience, whereas slimes are fairly tedious and full of monotony. I think it’d be a better option to nerf the “overpowered” slime cores or refactor their rewards over making the department a tedium

**Comment by Poojawa** [comment]
If the rewards aren't good enough, then no-one would play it anyway. 

It's honestly a catch22 in terms of being balanced. This is the most painless (if slighty more risky player wise) of achieving the powerful rewards people play xenobio to get

as for real, it isn't hard to replace piping. Play engineering for a while and you'll understand.

**Comment by lightcat** [comment]
People play this game for fun not to work for something. I'd rather more innovative content was introduced or a new fun way was added not simply removing content.

I do not support this change.

**Comment by Poojawa** [comment]
You're more than welcome to introduce innovative content.

There is nothing being removed here except for a round start tool that should have been late game. Just like how the smoke machine shouldn't have been roundstart, but it was until just recently.

**Comment by lightcat** [comment]
It's not my job to provide you with content to fix your addition or pursue a change you want to see.
You have six people here saying that this is not the way to go but if you dont care thats fine. I'm not here to argue, only to provide feedback.
There isn't even an reason of why this change is happening in the first place.
**It's boring, it doesn't add or change anything in a fun way and it seems half assed.** I mean fuck is there even a way to kill slimes now or is cargo going to be ordering a lot of fire extinguishers. This change is the equivalent removing all wires around the station and saying "You don't get to enjoy power if you dont work for it."

Jay wanted more feedback on the merges and he has it.

**Comment by Poojawa** [comment]
No, this is the equivalent of removing the smoke machine from chemistry. Or starting on a station that you have to set up power in.

If anything is half-assed, it's the fact that this is available *at round start* that provides you with just point, click, and drop "Gameplay" which is even more boring. the point of slimes is that you're supposed to be raising them as a species and learning about them. While also being rewarded for your efforts with increasing power.

No-one has told me yet of why doing nothing to earn rewards is more fun than doing the job in a roleplaying game and earning rewards. 

Also, I didn't say it was your job, and it isn't my job either. But trying to infer that I'm supposed to be the one in charge of changing shit, but not anything you like, is one of those "feedback" topics I have no use for. You can't demand for me to do shit, then get upset when I do shit.

**Comment by Knightfall5** [comment]
It's more fun because it isn't tedious, boring gameplay. Not sure how many times we have to repeat this, but xenobiology was a barren wasteland before consoles. Hardly anyone ever used it for anything, least used part of the station. When this was merged in, Xenobiology was barely used. It was a barren wasteland. It's been used more in a single day with the consoles added back in than in the entire time consoles were removed.

**Comment by Poojawa** [comment]
Well yeah. Who wouldn't want to play with something that gets you things for little effort.

Why wouldn't you want to play a slot machine that won you money just for pulling the handle?

**Comment by Knightfall5** [comment]
Hey, if you want to remove this "no effort" xenobiology system with one that requires effort and is fun, go for it. But xenobio without consoles isn't fun, and I think the fact that hardly anyone does it without consoles is proof enough of that.

**Comment by Poojawa** [comment]
Except. this don't remove consoles forever. it just takes out out of roundstart. That's all this does.

**Comment by Knightfall5** [comment]
And who wants to hop into a new round hoping to do Xenobiology, but only after you've managed to research the console and build it? Apparently that list is pretty small.

**Comment by deathride58** [comment]
With this change, xenobio consoles can still be easily accessed via RnD, the board is available in sci protolathes after T4 stock parts have been researched

**Comment by silicons** [comment]
as a tg powergamer with 17k connections over three years i can tell you that back in the old days we did it by harm intenting the monkies and dragging them in and a fire extinguisher.
hell some of us were damn good at it too and can power through it far faster than most console noobs can.

and you know what? it lead to interesting situations 
like how the scientist would actually BEFRIEND (bet some of you didn't know you could do that) the slimes and use them as weapons against specific people? and how you can order slimes around like to follow you or attack certain things?

i'm not going to speak on whether this is the right design decision but it is definitely not impossible considering there were still a good number of us who kept getting op things from it just like everything else when it wasn't as powerful, and people keep forgetting that most things were powercreeped up slowly and slowly.

**Comment by Knightfall5** [comment]
Yeah, that's how we did it before there was a console. It definitely wasn't faster though, and it was pretty boring. And yes, from playing on Bay/Polaris/Virgo I know about slime befriending and the orders you can give them. Pretty useless if you aren't an antag, and no antag is gonna sit around in xenobio without consoles to make an army when they could be shooting someone, especially when the slime cores (LOOKING AT YOU SEPIA) are way stronger than the slimes.

**Comment by Poojawa** [comment]
You can't get slime cores if you're too lazy to breed slimes. waiting 30-45m to start means you've got dead slimes from starvation. 

The console should be an *upgrade* to doing things, not an alternative means. if the cloners came with everyone at round start pre-scanned and autoprocessed, would you agree that was overly simplifying the point of medbay?

**Comment by Knightfall5** [comment]
Giving everyone free infinite revives isn't exactly the same as a console in Xenobiology that speeds things up. If the things coming out of xenobiology are OP, they are still OP without the console. They are just boring and tedious to get now, meaning people have less fun.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -161,6 +161,8 @@
 	// Monkey cube
 	if(istype(O, /obj/item/reagent_containers/food/snacks/monkeycube))
 		var/obj/item/reagent_containers/food/snacks/monkeycube/cube = O
+		cube.visible_message("<span class='danger'>The monkey cube starts to fizzle and crackle, it's growing!</span>") //CITADEL CHANGE Adds delay to monkey cube growth
+		sleep(10) // no I don't feel like making a timer just for this. fuck off.
```
Comment (silicons) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:165]: sleeps are b-
oh
:(

**Review by silicons — DISMISSED** [review]
it'd be nice to put consoles at about t3 part level instead of t4 due to decreased rate of generation but oh well

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by GrayRachnid** [comment]
improve not remove

**Comment by ShadowDeath6** [comment]
what is even the original complaint about xenobio? If it's too fast, make the slimes split more slowly or need more monkeys upon splitting. If it's too easy to get good variants, make the good variants less likely to mutate. 
The consoles make xenobio a lot more accessible to players wanting to try it out, and this entire complaint about it just seems like a complaint of "I had to do it the hard way, so stupid xenobio players should have to do it too". 
There's no difficulty in either way, the by-hand way just takes longer because of how annoying it is to pull slimes and handle monkeys. That isn't more difficult, it's just more annoying. The only way it's made xenobio more unbalanced is because you don't need to spend extra seconds wasting time trying to pull things around and then blast them with an extinguisher.

**Comment by Amelia0010** [comment]
>You have six people here saying that this is not the way to go but if you dont care thats fine.

[They don't](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/5787)

**Comment by deathride58** [comment]
@GrayRachnid This PR does not remove the xenobio console. It just makes it require t4 parts in RnD for the board to construct the xenobio console.

@ShadowDeath6 the issue is that the console makes xenobio completely effortless. With the console, xenobio is just a point and click breeding game that requires zero thought or effort. This PR makes it so the console is a lategame RnD upgrade rather than a standard roundstart piece of machinery. The console also means that slime interactions have fallen into complete obscurity. An entire game mechanic (slime interactions and relationships) that's been insanely fleshed-out over the years is a complete mystery to almost all players due to the console making it so you don't have to interact with those mechanics.

**Comment by GrayRachnid** [comment]
Xenobiology is not made easy by the consoles, but by miners.

If you truly remember the pre-console Xeno days then you'll remember the lack of miners due to the asteroid being a boring place and nobody wanting to go there. Xeno always lacked plasma and purple was always a priority. If you're going to bring up old days of Xeno then consoles isn't what brightened them up, its lavaland. Consoles do nothing other than make boring gameplay actually less boring. I can guarantee that slime interactions will not be used as you imagine them to be used even if this change is actually merged, because people would be too busy trying to farm slimes as quickly as possible to /care/.

Not to mention, Xeno will probably be abandoned until the tech required to build the consoles is there, just like last time. So yes, you are REMOVING and not IMPROVING. You are making a chill job a boring and a very tedious one that was completely fine as it is. If you want to change xeno because BALANCE then adjust other numbers, similar to what Shadow suggested. 

There is absolutely nothing wrong with the way Xenobiology plays right now, it's a chill, mindless job that people enjoy doing when they just want to chill. No pressure and the chances of death are low. If THAT is your problem then you should completely remove the xenoconsole because if this /ever actually gets merged/ then people will just abandon Xeno until the tech is available unless you make it worthwhile to do xeno early.

**Comment by ShadowDeath6** [comment]
>insanely fleshed out
>"hello slime"

**Comment by ShadowDeath6** [comment]
>insanely fleshed-out

image 2

It's like 50 or so lines of code, and was also never actually useful to use when it was feeding by-hand outside of checking whether or not the slimes wouldn't attack you (even telling the slimes to eat was less efficient than drag + drop because that still existed) when you fed them, and even on the off chance that you were an antag willing to feed the slimes plasma bars or monkeys, it wouldn't matter because amassing an army of slimes gets 5 people with fire extinguishers on your ass immediately and the slimes became a nonissue

also, locking the xenobio console behind R&D means that nobody is going to do xenobio until it's unlocked. I can guarantee that nobody is going to waste their time dragging slimes through one another  when they could be doing something more productive for the 15 or 20 minutes until consoles are unlocked

**Comment by Knightfall5** [comment]
Slime interactions are completely and utterly meaningless. People know they exist, they just don't use them because slimes are about as useful as a moving sack of jelly.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Poojawa) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:165]: lol

**Comment by Poojawa** [comment]
Having this console is like giving botany a gardening bot at round start. It defeats the entire point of animal husbandry or taking care of plants. It's a chore to keep up and to carefully breed things you want and they're supposed to be very powerful as a result. Being able to stop time on demand is just, ludicrous but I'm not preventing people from getting that. 

if, for you, fun = free stuff with no effort. Then I'm sorry, but you're going to have to go play cookie clicker  at this point. 

If slime interaction is meaningless, then I'm happy to port over mechanics that makes them more useful and deadly. Probably increasing the chances of them shock stunning and absorbing people would be lethal enough.

**Comment by ShadowDeath6** [comment]
removing the consoles doesn't make it require effort though, it's just annoying

this isn't making xenobio any weaker, this is just making it more of a chore to get the strong stuff.

instead of making the job less enjoyable why don't you make the rewards less unfair garbage

**Comment by Poojawa** [comment]
Give me some ideas on how to nerf the rewards in a way that justifies having no danger in acquiring them, then.

**Comment by ShadowDeath6** [comment]
it's probably easier if you made known what problems you had with the job before asking people to come up with solutions to those problems

**Comment by Knightfall5** [comment]
There is no danger in acquiring them without the console. It is just boring.

**Comment by Poojawa** [comment]
The only problem I have with the job is how braindead easy it currently is to gain the rewards. Instead of nerfing the rewards to treat the symptoms of the problem, I'm removing the problem that cause the symptoms. 

That's how fixing stuff works.

**Comment by Knightfall5** [comment]
So you leave the OP stuff like Sepia time stop in, make the job super boring, and expect what to happen exactly? Powergamers gonna powergame, and the people who used to do xenobiology for fun now won't play it anymore.

**Comment by Poojawa** [comment]
You've not bothered to give me any solutions to nerf rewards. and even when I ask, I don't get anything. 

All I gather from the negative feedback is:
Console = Fun and engaging, and important cornerstone to being able to do xenobio at all.
No Console = Xenobio might as well be deleted because it's super boring. Having to play the department is "annoying" because you have to manage your character in addition to slimes and monkeys, rather than just teleporting them around. Slimes are suddenly "worthless" for the effort it takes to get them and anyone who plays it is going to get bored halfway through.

We can sit and rehash your points over and over, I guess. But I don't believe your definition of 'Fun' is the only feedback required to deny this change. Please provide other solutions and feedback.

**Comment by ShadowDeath6** [comment]
Xenobio as a job will never be fun and engaging because it's nothing but sitting and waiting for slimes to grow. The only point of xenobio is the endgame rewards. The console is not and won't be fun and engaging, because it's part of xenobio. It's not more dangerous or harder to shove slimes and monkeys around, but it is more annoying to pull slimes through one another. You're not making the job harder, you're making a boring and dull job more annoying and doing nothing about the balance of the job itself.
You aren't treating any problem besides the amount of people who feel like playing it. If you're going to go down the route of making the job more time consuming and not difficult, make it so that you don't get the slime processor roundstart either, because surgery on slimes is also braindead simple and the slime processor is literally just free tech that makes the boring parts of xenobio less time consuming.

**Comment by GrayRachnid** [comment]
Also, shadow mentioned some alternatives a few comments up so you can't say there aren't any.

**Comment by Knightfall5** [comment]
Since you suddenly want feedback on how to nerf rewards, get rid of Sepia slime's ability to stop time. Most OP thing from the entire job gone. Maybe make it so you can't turn yourself into a changeling too. Just don't get rid of the damn console.

**Comment by Nightspark43** [comment]
Removing the console is pretty much exactly the same as removing the slime and monkey processors, all it does is make the thing more tedious and unworthwhile, while removing them won't break the concept, it's really not necessary to remove them, all removing them would do was make people not want to do it until they made it easier for themselves halfway through a round.

Your rampant overnerfing of systems that didn't need nerfed as hard as they were is honestly becoming a pain in the ass, 1200 points is BS, 1900 is on the low side, but fine, removing the xenobiology consoles is a major annoyance to anyone who wants to actually do xenobiology, what's the point in it besides unneeded balance, if you want balance, nerf sepias, those are things that actually might need nerfs.

**Comment by Poojawa** [comment]
and you were doing so well in giving feedback Night.

**Comment by Nightspark43** [comment]
I can only take so much.

**Comment by Poojawa** [comment]
That makes two of us, then.

**Comment by Knightfall5** [comment]
Well, those _certain_ individuals turned out to be a pretty big majority, so can we go ahead and dump this in the bin since it's been four days now?

**Comment by Poojawa** [comment]
60-70 pop average during peak times

10% of people voted no with the only reason is 'fun removal' 
"pretty big majority"
🤔 

I didn't know this was Trump election math

**Comment by Amelia0010** [comment]
>60-70 pop average during peak times

>10% of people voted no with the only reason is 'fun removal'
>"pretty big majority"
>🤔

>I didn't know this was Trump election math

And only 3 people voted in favour. Either you don't count "didn't vote" votes, or you put them in favour of the status quo. Either way, you can't in any way use the argument that because not everyone voted you get a perogative to go ahead.  70% of all the people who voted voted in favour of keeping it like it is.

I believe you're the one trying to use bad election maths, not anyone else.

**Comment by Amelia0010** [comment]
And just to make it clear, I don't personally care about this change, I care about you repeadetly trying to force changes in that the majority of the people who care enough to vote dislike. Keeping a PR open for excessive periods of time because it doesn't have a majority of the server pop's downvotes is idiotic, especially when it has a 70% downvote rate.

It's not **just** a pretty big majority, it's bigger than a supermajority. And comparing it to the server's pop at highpop is not usefull for anyone.

**Comment by Poojawa** [comment]
the fact that it's being called to vote at all is more so, but here we are.

**Comment by Amelia0010** [comment]
Would you rather we say that 70% dislike it? Or that 70% thumbs-downed it?

**Comment by yourdoom9898** [comment]
We have either #5893 that does just the map updates, which hasn't had a disagreeing opinion on it, or we have this PR, which has shown a heavy amount opinions against it. Any reasonable person would just pick the #5893 by this point, since we have made it past the three day waiting period.

**Comment by nichlas0010** [comment]
>Any reasonable person

There's your issue

**Comment by Poojawa** [comment]
obviously merge after #5951 didn't want to fuck with a billion conflicts so I just reset from that branch.

**Comment by Poojawa** [comment]
good to go.

**Comment by Amelia0010** [comment]
it has 7 downvotes and 4 upvotes, why would anyone merge this against the wishes of the people who care enough to vote.

**Comment by Poojawa** [comment]
Because this isn't a democracy, my dude.

**Comment by yourdoom9898** [comment]
Are you really willing to start another shitfest over a change that has been reverted once, and argued about the second time?

**Comment by Knightfall5** [comment]
Actually it is a democracy now, didn't you get the memo? PRs are closed or merged based on the community's opinion, because _certain_ individuals can't be responsible with their power. I suggest you go read the pinned message from Jay in the dev channel on discord, before you end up breaking the rules. Again. Again. Again.

**Comment by ZQFarnzy** [comment]
I'm hesitant. I know it's possible to do xenobio without consoles, but removing these with no compromise or positive change is just removing fun from the game, and nerfing science. I oppose this change, what little the words of a non-coder mean here.

**Comment by deathride58** [comment]
@ZQFarnzy Again, this PR doesn't outright remove the xenobio consoles. It just makes xenobio consoles 100% RnD exclusive

**Comment by Amelia0010** [comment]
>[05:07] Jay: As of now, @everyone, to force people to voice their concerns and/or opinions on a pull request, at least one person from the project team must review and approve the PR before it can be merged. After three day period, those PRs will be merged or closed base on the opinions, resolutions, or lack of after. Mirror PRs are subject to exclusion from this rule.

It's been 5 days and there's a supermajority in favour of closure.

**Comment by GrayRachnid** [comment]
Changing my vote just so I can watch new people to xenobio attempt to learn this on their own and die because the wiki doesn't support the manual way. :+1:

**Comment by Poojawa** [comment]
With a preferred alternative actually hashed out instead of petty bickering from people complaining about fun removal, I'll close this.

**Pull request closed without merging**